parm/config/gfs/config.resources

```diff
@@ -1045,13 +1045,16 @@ case ${step} in
         ;;
       "C384")
         export npe_eupd=270
-        export nth_eupd=8
+        export nth_eupd=5
```

Comment:
@HenryWinterbottom-NOAA 
Is this change necessary? This will impact all machines except the ones noted in the if block below.
Also, in the if-block, Hera is reset to 8.  8 is the develop version.
Is the only change needed here on line 1056?  For Hera, `npe_eupd=80`?

Reply:
No, thank you for catching my oversight. Updating only line 1056 results in

```
<task name="enkfgdaseupd" cycledefs="gdas" maxtries="&MAXTRIES;">

        <command>/scratch1/NCEPDEV/da/Henry.Winterbottom/trunk/global-workflow.gwdev_issue_2454/jobs/rocoto/eupd.sh</command>

        <jobname><cyclestr>x002_gwdev_issue_2454_enkfgdaseupd_@H</cyclestr></jobname>
        <account>fv3-cpu</account>
        <queue>batch</queue>
        <partition>hera</partition>
        <walltime>00:30:00</walltime>
        <nodes>16:ppn=5:tpp=8</nodes>
        <native>--export=NONE</native>
        ```
        
        Which is consistent with @wx20jjung configuration:
        
        ```
        <task name="enkfgdaseupd" cycledefs="gdas" maxtries="&MAXTRIES;">

        <command>/scratch1/NCEPDEV/da/Henry.Winterbottom/trunk/global-workflow.gwdev_issue_2454/jobs/rocoto/eupd.sh</command>

        <jobname><cyclestr>x002_gwdev_issue_2454_enkfgdaseupd_@H</cyclestr></jobname>
        <account>fv3-cpu</account>
        <queue>batch</queue>
        <partition>hera</partition>
        <walltime>00:30:00</walltime>
        <nodes>16:ppn=5:tpp=8</nodes>
        <native>--export=NONE</native>
        ```
        
        I pushed the correction.